lib/casserver/views/layout.erb

```diff
@@ -66,7 +59,7 @@
NFngg3rNsN4t7hr1LfnxRV380L90GPGgeRORU4sgY4i4tvJHDEjt8efhfpLK1S12MdJkR6YbjST0S+IM0i/71FI68h9kUSW6cizyWxWxxc/KuJ7/2COiA1khg+YtsHnGtiO5SfU9L2Y0ncTIr8ZWAFiX+viW+qyLtjV83rLCxT2yZG1bd9IbXLH1P3RqQC/yyuTwp3c7qBvrg+YfUSAcbi+kasPm6328DRZA8eD9Zpd5FNhVPU/+aUqu9BXfZHGlQi8ltFAu/lIv93SZlTkZtlTTA/+ytiu1OR58/pcOVrFhfwFhJ/sYOLv5r4Jsezt5H3lzuMc5WJ3c3E7i5JzEwOV/4BXKroaUFjGaK5QarIO6tnfPP9CWF1VW4AxuM16v5oo9kwBL/Bg2P0UcCFcT3NiOCK6QizXpYzgtsNM5AFIv4uCHwxzAPULUVT1BZvSF3uKdY+aPtA5KQOY0Wx98Dc4Xh0KruvoJRd6EzkWxU9T7yM85yiZx52EjvsuaIWF4T25F9X9J7xPAP4N/bw38GGJ2XcbvPCFxM/Y6YB5xGfkuPZklg3EtuXTOw2Gmwwrz1CEpOK/CFFv2tdQcA5AE0/Wok8ebe7npTRLvBNFfnqSONZy3MDmz4sKFMPlU3VBAW9o9oHw5UD0cZ++N2cCNfnWoh4QCT4747iSn+LN/jasjGFxzdeUa7Vo9dQj/3RhmYQPRNxMNLITpQ1sTrfRDYsW7pvRZRCtMNvbYjPKHKcFOEHo1PRx85JOhR9sYnGx2yI1fAmySLi20SO/5Uc/5Mk1jBedtL2uEB45STe3y3qJ0ohmmE1D6Aif8twbDmJRdOmqSJfUzGdtZVWz0kEYiqOw1FYe+F3IlynriuItm0sKGc7Sqq0YUEiA0KPa/nxWra5qGLI+sKybhwqJbjAsTfvCdWqLXmhGW96g6zF92Xe8YXY+rwwa8qdwIbn5GRC5Lha+S8LIYry5HbAW2+9xcVO8g0lNd8uWiXE2/kR4wVW9B7eLOZTpD53k+N08l1oUUcnIn/WIhYV+ZuGY5eQ80TKDMdTRb4XBLYvUv9eggscBXYQhiyr80PqktGT+mGc3rx1ckVuTwZFDh8uPwj6NbhzHE8KHOuDPXprflh9ccwVzaty/GylgDdzSlmrJ8DKc3xs5Hjp5iwqclwSx56qyQAKDpf/sffeRezXKvp4PR0ix4kejjFlk1nK/SQGLgbhfpUAx6/oWZtNxO6nkphBYvsREZlxgcxOBoiKvJP7tRIwNz/LIpZM5Mh1pKwuw7EPDO9refI649gXBb6luEJdOqZAbVGCKvTk6o7Sam1MnkjzmDyOY/JQ3WLBHAAEDvUoqlBf8ZSobflRtb1oytj2ogq2tWhKCtuAA8VV7O78yewi+TZbKvL7LBoZx9XtxMds0cYoclykwdXBWkVPO1YBD5I4onGkDBzT09z2UfK32ftWZX1A7MP8/dnk/Stt1JOK/BaH5+lE5MhiUt7j/P21hvc0kc9NCivBx8ArvX71VU+p+iEIHNN6yWPxDGRXROyFYcr24ip1Za4fBB5RO0oq1a5gza24GvpJoCaFLkCFsfkLRRE21JnIrfLkyA+Ij9mY1M5iEO1hrXoyI3SS6JQGSeyZirhdugzv7bRZTyrymx2c40BEjqvQdNEO78jGYZYm8rVfiAsE3hWI7VvlDe5rywtr4sIxsYOc+YDy5Eb6QOBd/ljvW0qwfx0IvLO0Ut0dqMHtBgt6te0GX0aNNLGFKHJnE087Iq8kPktN7OzkyWtIrMMmdiJob9uh6EOLrQI2AHuJ/QpJ7JFK6rAEhxM3EX9Z2lAm8nTmyUUiF7Urnvt2hYi8z9iTx8P1KzYUVV6+2V+9FHpOmqvO2IqnoRef01FcNaptYmRXR0mVujtYo+XRoR5vJ/i8gJKhFU/kduIz18TO7mLQqyTeYzbqUEF8cFKFPRjm7c8VgD6i3Yai/SFJnjPYoeCNE1ycHNvdB386RI48KTjfFJGn7ELsC9Ut2xWsY9tqrmTxssZpiWDtKdm7AswHwfrRt7OkinWUVm7uDsQ0gXObdO9dMVstS4Jj0zjxMVvcsCty0fZYr0U9XiL237MpkiXE73GJLe0NjcxzIEwqcqcbzwYqcoSub6SI/GCK4IK1CxMwxt1VNYN1l09lIHK0yfguRPRJhOtYd6CWdZXWYFpxA/FNt8hfV/RVwlkGfqboK6H/FDQW5riHmsR2sqxP03bbFPPUXDax7XUgkhLii/WS7U7sNBGJVdpQJvJlgjam4FAIN5TRbzU5FTk9X/OeHHhdSxWG61lPtBE3cMmyJmndT66hb/E9L6FPTo2+mdhP7oRKSWwnIscVQ5rjxeGBaH/2n4ndbxwIDmkl/rJ04mzBOTvdOOYkT04ZSWI5FTnygCCuJnLhN4M0ocsFjmTqm0EFAt+T/WYQXQ62y0rFelmfpvdkS99Io6AcmsEYZ0MIVlwjiGFWt2yB7dUOyxN9UAYq8tHk+HabdWijsVEk/wff8ZzGnirSHyzkMz95vJ0tVPS92/MkLAD+pui30ojNhp3DY+MCDy6tmw1rjOAHByeiuDCEk7vnyfFKRR82YZ0wH3ybQ8EhuFEMl/ab+XnjAtgEiT1uTFvEzwUnxU6/sVTP6yprX1F7PypoM5zsPs3bFOt/j8065PC487nvkyiSGwSCGlTf1j8WncqWl5SzSzw5zDM+16kQTje4XK8ILrJLmkCRDPrnrhyA+cMhIObLZyM80v0rLl9BZMMDZFA8QQvnDyfKprEHCwLsXPcxcS6EpFAG9bMQtS9OlE/XnoXIxo6VPWDI5SuIUSyyp9ricv0P42fwU20/jzay1aWV2ph8dM6gG5e7ZBAqFrvPJ2+On2HPJz8SqWfd8LtY8bFR7vMQXQyIBDMo/9NEf6SBbQvG2KRcL8tyRe5iwEw0g+5/BrkidzFDJpxB9d/fXJG7mGFHQIPi/3i6Incx438NLDZ8B0Z7RAAAAABJRU5ErkJggg=="></a>
```

Comment:
On Fri, Dec 11, 2015 at 07:44:13AM -0800, Brian Sadler wrote:
> This has an un-needed redirect since we serve everything over ```https```

might as well change them all!